javatests/arcs/android/devtools/DevToolsMessageTests.kt

```diff
@@ -0,0 +1,124 @@
+package arcs.android.devtools
+
+import arcs.android.devtools.DevToolsMessage.Companion.ACTOR
+import arcs.android.devtools.DevToolsMessage.Companion.CLOCK
+import arcs.android.devtools.DevToolsMessage.Companion.KIND
+import arcs.android.devtools.DevToolsMessage.Companion.MESSAGE
+import arcs.android.devtools.DevToolsMessage.Companion.MODEL_UPDATE_MESSAGE
+import arcs.android.devtools.DevToolsMessage.Companion.OPERATIONS
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_ID
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_OP_MESSAGE
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_TYPE
+import arcs.android.devtools.DevToolsMessage.Companion.TYPE
+import arcs.android.devtools.DevToolsMessage.Companion.UPDATE_TYPE
+import arcs.android.devtools.DevToolsMessage.Companion.VALUE
+import arcs.core.crdt.CrdtData
+import arcs.core.crdt.CrdtEntity
+import arcs.core.crdt.CrdtOperation
+import arcs.core.crdt.CrdtSingleton
+import arcs.core.crdt.VersionMap
+import arcs.core.data.util.ReferencablePrimitive
+import arcs.core.storage.ProxyMessage
+import arcs.core.util.JsonValue
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class DevToolsMessageTests {
+
+    @Test
+    fun storeOperationMessageTest() = runBlockingTest {
```

Comment:
IIRC there aren't any suspend methods being used in these tests, right? `runBlocking`/`runBlockingTest` is only necessary for tests that end up calling suspend functions.

That said - it doesn't hurt to have this here, so keeping it is no big deal.